Formula/fastfec.rb

```diff
@@ -0,0 +1,27 @@
+class Fastfec < Formula
+  desc "Extremely fast FEC filing parser written in C"
+  homepage "https://github.com/washingtonpost/FastFEC"
+  url "https://github.com/washingtonpost/FastFEC/archive/refs/tags/0.0.3.tar.gz"
+  sha256 "13317652775d9734af52440d8b3fb3a0c759f94345b84ad7d2c80efdc18d5e0d"
+  license "MIT"
+
+  depends_on "pkg-config" => :build
+  depends_on "zig" => :build
+  depends_on "pcre"
+  uses_from_macos "curl"
+
+  def install
+    ENV["ZIG_SYSTEM_LINKER_HACK"] = "1"
+    system "zig", "build", "-Dvendored-pcre=false"
```

Comment:
```suggestion
    ENV["ZIG_SYSTEM_LINKER_HACK"] = "1"
    args = ["-Dvendored-pcre=false"]
    args += ["--search-prefix", Formula["curl"].opt_prefix] if OS.linux?
    system "zig", "build", *args
```
This might fix the Linux build failure.

Reply:
Good timing! Just pushed something that looks fairly identical. Let me know if you prefer your version

Reply:
Either is fine, but let's use `shared_library` to install `libfastfec`.

Reply:
Oops, I forgot to pull that commit locally when I accepted your suggestion through GitHub's UI. Added back in